openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/services/screen.service.ts

```diff
@@ -1,9 +1,10 @@
 import { Injectable, Type, ComponentFactoryResolver, ComponentFactory } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
+import { Observable, throwError } from 'rxjs';
 import { IScreen } from '../../shared/components/dynamic-screen/screen.interface';
 import { PersonalizationService } from '../personalization/personalization.service';
 import { DiscoveryService } from '../discovery/discovery.service';
+import { catchError } from "rxjs/operators";
```

Comment:
Caught by the linter
```suggestion
import { catchError } from 'rxjs/operators';
```